api/tests/routes/native/v1/offers_test.py

```diff
@@ -1930,6 +1982,22 @@ def test_anonymize_author_of_chronicles(self, client):
         chronicle = response.json["chronicles"][0]
         assert chronicle["author"] is None
 
+    def test_future_offer(self, client):
+        offer = offers_factories.OfferFactory()
+        publication_date = datetime(2050, 1, 1)
+        _ = offers_factories.FutureOfferFactory(offerId=offer.id, publicationDate=publication_date)
+
+        offer_id = offer.id
+        nb_queries = 1  # select offer
+        nb_queries += 1  # select stocks
+        nb_queries += 1  # select mediations
+        nb_queries += 1  # select chronicles
+        nb_queries += 1  # select futureOffer
+        with assert_num_queries(nb_queries):
+            response = client.get(f"/native/v2/offer/{offer_id}")
+            assert response.status_code == 200
+            assert response.json["publicationDate"] == "2050-01-01T00:00:00Z"
```

Comment:
```suggestion

        assert response.status_code == 200
        assert response.json["publicationDate"] == "2050-01-01T00:00:00Z"
```
(vraiment détail) désindenté ça marche aussi et comme ça on est cohérent avec le reste de la codebase